core/sql/common/NAType.h

```diff
@@ -562,7 +562,8 @@ class NAType : public NABasicObject
 				 short displayCaseSpecific = 0);
 
   short getMyTypeAsText(NAString * outputStr, 
-			NABoolean addNullability = TRUE) const; // output
+			NABoolean addNullability = TRUE,
+                        NABoolean addCollation = TRUE) const; // output
```

Comment:
Looks like the "// output" comment migrated to the wrong place. Maybe it belongs on "NAString * outputStr"? Or we could just delete it.

Reply:
thanks, will fix that.